addons/survey/static/tests/tours/survey_roaming_mandatory_questions.js

```diff
@@ -4,7 +4,6 @@ import { registry } from '@web/core/registry';
 
 registry.category('web_tour.tours').add('test_survey_roaming_mandatory_questions', {
     test: true,
-    url: '/survey/start/853ebb30-40f2-43bf-a95a-bbf0e367a365',
```

Comment:
I guess these are not necessary because we specify the url in python?

Could be mentioned in commit message

Reply:
Sure, would just specify that it was already redundant in commit message so we don't wonder what it has to do with the rest of the diff